pkg/manager/manager.go

```diff
@@ -74,18 +74,31 @@ func (m *manager) SetupVF(conf *types.PluginConf, podifName, cid string, netns n
 		}
 	}
 
-	// 4. Change netns
+	// 4. Set MTU
+	if conf.MTU != 0 {
+		prevMTU := linkObj.Attrs().MTU
+		if err = m.nLink.LinkSetMTU(linkObj, conf.MTU); err != nil {
+			return "", fmt.Errorf("failed to set MTU on VF %s: %v", linkObj.Attrs().Name, err)
+		}
+		log.Info().Msgf("VF link %s MTU set to %d", linkObj.Attrs().Name, conf.MTU)
+		if err != nil {
```

Comment:
Why we need this check?

Reply:
Left over from last iteration, fixing.
Thanks.